source/math/FloatPoint2DVectorList.ooc

```diff
@@ -87,6 +87,54 @@ FloatPoint2DVectorList: class extends VectorList<FloatPoint2D> {
 		}
 		result
 	}
+	offset: func (offset: FloatPoint2D) -> This {
```

Comment:
I was thinking about maybe calling it shift, but that is used in a different way in FloatVectorList.